src/test/java/lambda/part2/exercise/Exercise2.java

```diff
@@ -0,0 +1,99 @@
+package lambda.part2.exercise;
+
+import lambda.data.Person;
+import org.junit.jupiter.api.Test;
+
+import java.util.function.Predicate;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.is;
+
+@SuppressWarnings({"ConstantConditions", "unused"})
+class Exercise2 {
+
+    @Test
+    void personHasNotEmptyLastNameAndFirstName() {
+        // TODO предикат Person -> boolean, проверяющий что имя и фамилия человека не пусты
+        Predicate<Person> validate = person -> !person.getLastName().isEmpty() && !person.getFirstName().isEmpty();
```

Comment:
u can get null pointer exception, fix it

Reply:
fixed (4 times)